LaunchAtLogin/LaunchAtLogin.swift

```diff
@@ -1,21 +1,156 @@
+import Combine
 import Foundation
 import ServiceManagement
 
-public struct LaunchAtLogin {
-	private static let id = "\(Bundle.main.bundleIdentifier!)-LaunchAtLoginHelper"
+public class LaunchAtLogin {
+    public static let observable = LaunchAtLogin()
+    public static let kvo = observable.kvo
 
-	public static var isEnabled: Bool {
-		get {
-			guard let jobs = (SMCopyAllJobDictionaries(kSMDomainUserLaunchd).takeRetainedValue() as? [[String: AnyObject]]) else {
-				return false
-			}
+    @available(macOS 10.15, *)
+    public static var publisher = observable.publisher
 
-			let job = jobs.first { $0["Label"] as! String == id }
+    public static var isEnabled: Bool {
+        get { observable.isEnabled }
+        set { observable.isEnabled = newValue }
+    }
 
-			return job?["OnDemand"] as? Bool ?? false
-		}
-		set {
-			SMLoginItemSetEnabled(id as CFString, newValue)
-		}
-	}
+    private let id = "\(Bundle.main.bundleIdentifier!)-LaunchAtLoginHelper"
+
+    public lazy var kvo = LaunchAtLoginKVO(launchAtLogin: self)
+
+    @available(macOS 10.15, *)
+    private lazy var _publisher = CurrentValueSubject<Bool, Never>(isEnabled)
+    @available(macOS 10.15, *)
+    public lazy var publisher = _publisher.eraseToAnyPublisher()
```

Comment:
Why are you erasing it? Then the user wouldn't be able to actually get the current value from the publisher.

Reply:
It is erased so the user will not be able to `send` custom values. A current value will be emitted on subscription, so the user will get it.

Reply:
Ah ok. 👍🏻